tests/acceptance/features/apiSpaces/quota.feature

```diff
@@ -139,3 +139,13 @@ Feature: State of the quota
     When user "Brian" tries to create a space "new space" of type "project" with quota "51" using the Graph API
     Then the HTTP status code should be "400"
     And the user "Brian" should not have a space called "new space"
+
+
+  Scenario: user can restore a file even if there is not enough quota to do so
```

Comment:
might separate it from trashbin restore
```suggestion
  Scenario: user can restore a file version even if there is not enough quota to do so
```

Reply:
test applies to both action `quota` and `restore`
I would leave test here to see enogth quota coverage. 

To restoring feature I can add simple test where I restore file from trash